ci-operator/step-registry/baremetal/lab/agent/conf/disconnected/proxy/baremetal-lab-agent-conf-disconnected-proxy-chain.yaml

```diff
@@ -6,7 +6,6 @@ chain:
     - ref: baremetal-lab-pre-load-balancer
     - ref: baremetal-lab-pre-dns
     - ref: baremetal-lab-pre-firewall
-    - chain: mirror-images-by-oc-adm
```

Comment:
It should be replaced by `ref: ipi-install-rbac`. See my trying: https://github.com/openshift/release/pull/43098

Reply:
hmm, right...

Reply:
I guess we can work on your PR and closed mine.

Reply:
Yeah, just wondering why other workflows seem not to run the ipi-install-rbac step here, and they seem working fine (they install the image)... If we had some reasoning to not include the step or we just missed it in each workflow under baremetal/.

Reply:
make sense,

Reply:
the pull secret used to install the cluster is already able to pull the release image (cvo mirrored into the ephemeral namespace in prow:

```
└ $ oc -a ps.json adm release info registry.build05.ci.openshift.org/ci-op-zrw1chxj/release:latest      
Name:           4.14.0-0.nightly-2023-09-02-132842
```

it's also possible to pull the component images:

```
└ $ oc -a ps.json adm release info --image-for=tests registry.build05.ci.openshift.org/ci-op-zrw1chxj/release:latest
quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7666527a0770cb2a168dbd709a2c476458424d7dff30de7e3fe207430bd036d0
└ $ oc -a ps.json image info quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7666527a0770cb2a168dbd709a2c476458424d7dff30de7e3fe207430bd036d0
Name:        quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7666527a0770cb2a168dbd709a2c476458424d7dff30de7e3fe207430bd036d0
```

Not sure why that is needed for other cluster profiles, but equinix-ocp-metal-qe seems not.

Is there any specific case I'm missing to justify broadening the permissions on the ephemeral ns? If yes, I'll change for all the other workflows under baremetal/... otherwise, it should be fine.